include/organizations_metadata

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
+# Prowler - the handy cloud security tool (copyright 2018) by Toni de la Fuente
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+# Gets account details with a given ACCOUNT_ID.
+# Prowler requires organizations:ListAccounts* and organizations:ListTagsForResource
+# in the management account in order to get that data. SecurityAudit managed policy includes them.
+
+# Account Tags are in json format with comma, however they are converted to Base64 
+# in order to avoid breaking the CSV or JSON. To use them a post-processor is needed.
+
+get_orgs_account_details(){
+    # -O to get info from Orgs management account needs also (-A) and (-R)
+    if [[ $MANAGEMENT_ACCOUNT_ID ]];then
```

Comment:
Why we need this conditional?

`$MANAGEMENT_ACCOUNT_ID` is checked before calling this function.